sky/clouds/cloud.py

```diff
@@ -466,6 +466,15 @@ def check_disk_tier_enabled(cls, instance_type: str,
             exceptions.NotSupportedError: If the disk tier is not supported.
         """
         raise NotImplementedError
+    
+    @classmethod
+    def check_quota_not_zero(
+        cls,
+        region: str,
+        instance_type: str,
+        accelerators: Optional[Dict[str, int]] = None,
+        use_spot: bool = False) -> bool:
+        return True
```

Comment:
Great to have a docstrings here. [This](https://github.com/skypilot-org/skypilot/blob/bbdd5b1512aae7cb1172b91b2b74cd8b0187f002/sky/clouds/cloud.py#L309-L352) is a good example.